cla_backend/translations/cy/LC_MESSAGES/django.po

```diff
@@ -482,7 +482,9 @@ msgid "For example admissions or exclusions"
 msgstr "Er enghraifft addefiad neu gau allan"
 
 #: apps/diagnosis/data/checker-graph.graphml.tpl:380
-msgid "You want to protect yourself or your children against domestic violence"
+#, fuzzy
```

Comment:
you will need to remove #,fuzzy and clear the translation or rebuild po files without fuzzy translations